lib/debezium/types.go

```diff
@@ -99,6 +100,27 @@ func toInt64(value any) (int64, error) {
 	return 0, fmt.Errorf("failed to cast value '%v' with type '%T' to int64", value, value)
 }
 
+// ShouldSetDefaultValue will filter out computed fields that cannot be properly set with a default value
+func (f Field) ShouldSetDefaultValue(defaultValue any) bool {
+	switch castedDefaultValue := defaultValue.(type) {
+	case nil:
+		return false
+	case *ext.ExtendedTime:
+		return !castedDefaultValue.Time.IsZero()
+	case string:
+		if f.DebeziumType == UUID && castedDefaultValue == "00000000-0000-0000-0000-000000000000" {
```

Comment:
Could we use `uuid.Nil` here instead of hardcoding, to make sure we have the right number of 0s?

Reply:
Sure, I was debating doing `uuid.Nil.String()` or hardcoding the literal value. I'll add `uuid.Nil` back so it's more clear